spanner/spanner_samples.rb

```diff
@@ -968,6 +968,38 @@ def delete_using_partitioned_dml project_id:, instance_id:, database_id:
   # [END spanner_dml_partitioned_delete]
 end
 
+def update_using_batch_dml project_id:, instance_id:, database_id:
+  # [START spanner_dml_batch_update]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  require "google/cloud/spanner"
+
+  spanner = Google::Cloud::Spanner.new project: project_id
+  client  = spanner.client instance_id, database_id
+  insert_statement = "INSERT INTO Albums "\
+    "(SingerId, AlbumId, AlbumTitle, MarketingBudget) "\
+    "VALUES (1, 3, 'Test Album Title', 10000)"
+  update_statement = "UPDATE Albums "\
+    "SET MarketingBudget = MarketingBudget * 2 "\
+    "WHERE SingerId = 1 and AlbumId = 3"
+  row_counts = Array.new
+  row_count = 0
+
+  client.transaction do |transaction|
+    row_counts = transaction.batch_update do |b|
```

Comment:
Does `rows` fit better here as a name?

Reply:
This variable contains an array of row counts updated or inserted by each DML transaction that is executed so I'd prefer to leave it as row_counts if that's acceptable.

Reply:
I've renamed the variable previously name row_count to statement_count to make it more accurately named.

Reply:
* previously named row_count